import-export-cli/integration/config.yaml

```diff
@@ -19,5 +19,4 @@ admin-rest-api-version: v3
 devportal-rest-api-version: v2
 publisher-rest-api-version: v3
 devops-rest-api-version: v0
-apictl-version: 4.1.0
-
+apictl-version: 4.1.0
```

Comment:
Any reasoning behind this change? If not, shall we add the new line at the EOF?

Reply:
+1

Reply:
The change which Benura did is correct. Previously, there were two new lines at the end of the file. But after Benura's change, we will have only one new line at the end.
In GitHub, if there are no new lines at the end of the line, it will be shown by marking in red like below. For Benura's change, GitHub has not highlighted something like that, which means it's correct.

![image](https://user-images.githubusercontent.com/25246848/183815118-4df2846e-4c51-419e-8089-fa9a8b83dbd1.png)

Reply:
My mistake, we just need a single new line at the EOF